Data/SQLite/src/SessionImpl.cpp

```diff
@@ -205,16 +205,20 @@ void SessionImpl::open(const std::string& connect)
 
 void SessionImpl::close()
 {
-	if (_pDB) {
+	if (_pDB) 
+	{
 		int result = 0;
 		int times = 50;
-		do {
+		do 
+		{
 			result = sqlite3_close(_pDB);
-			if (result == SQLITE_BUSY) {
+			if (result == SQLITE_BUSY) 
```

Comment:
So, if I want to close session correctly, then I must to have list of all statements, and into close method all statements must be destroyed before sqlite3_close. And every statement  into destuctor must erase itself from session which owned statements-list. Am I right?